__tests__/index.test.js

```diff
@@ -105,7 +87,64 @@ describe('/api/reviews/:review_id', () => {
             .get("/api/reviews/777")
             .expect(404)
             .then(({ body }) => {
-                expect(body.msg).toBe("Review 777 could not be found");
+                expect(body.msg).toBe("Not Found");
+            });
+    });
+});
+
+
+describe('GET /api/reviews/:review_id/comments', () => {
+    test('GET:200 responds with an array of comments for the given review_id of which each comment should have the following properties:- `comment_id`- `votes`- `created_at`- `author` which is the `username` from the users table- `body`- `review_id`comments should be served with the most recent comments first',
+        () => {
+            return request(app)
+                .get('/api/reviews/2/comments')
+                .expect(200)
+                .then((response) => {
+                    const { comments } = response.body;
+                    expect(comments.length > 0).toBe(true);
+                    comments.forEach(comment => {
+                        expect(comment).toMatchObject({
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String)
+                        });
+                    });
+                    expect(comments[1]).toEqual(
+                        expect.objectContaining(
+                            {
+                                comment_id: 1,
+                                body: 'I loved this game too!',
+                                review_id: 2,
+                                author: 'bainesface',
+                                votes: 16,
+                                created_at: '2017-11-22T12:43:33.389Z'
+                            }
+                        )
+                    );
+                    expect(comments).toBeSortedBy('created_at', {
+                        descending: false,
+                    });
+                });
+        });
+    test('GET:404 sends an appropriate and error message when given a valid but non-existent id', () => {
+        return request(app)
+            .get('/api/reviews/777/comments')
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe('Not Found');
+            });
+    });
```

Comment:
We're missing tests here. What happens with an invalid id? What happens with a valid id that returns no comments?